scripts/demo-rn-bloat-bundle-test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+
+echo "Bloating the demo project bundle"
+run_f "npm run bloat-bundle"
+
+run_f "npm run build:ios-release"
```

Comment:
It looks like it's specifically run on ios - what about android? Since the script is being triggered from android/ios scripts they could pass in the platform as an argument

Reply:
correct! great catch